### PR TITLE
json parsing fix changes

### PR DESCRIPTION
**Pull request by manoj-freyr**

### CODE REVIEW

File: src/rvsliblogger.cpp [hunk 1]
```diff
@@ -347,15 +348,22 @@ int rvs::logger::JsonActionStartNodeCreate(const char* Module, const char* Actio
     rvs::logger::JsonStartNodeCreate(Module, Action);
   }
   isfirstrecord_m = true;
-  std::string row{RVSINDENT};
+  std::string row{newline};
+  if (isfirstaction_m){
+    isfirstaction_m = false;
+  } else{
+     std::cout << "not first ction"<<std::endl;
```
Comment (jkottiku) [inline src/rvsliblogger.cpp:355]: "action" typo

File: src/rvsliblogger.cpp [hunk 2]
```diff
@@ -347,15 +348,22 @@ int rvs::logger::JsonActionStartNodeCreate(const char* Module, const char* Actio
...
+  row += std::string(RVSINDENT);
   row += std::string("\"") + Action + std::string("\"") + kv_delimit + list_start + newline;
   std::lock_guard<std::mutex> lk(json_log_mutex);
   return ToFile(row, true);
 }
 
 int rvs::logger::JsonActionEndNodeCreate() {
   std::string row{RVSINDENT};
-  row += list_end + std::string(",");
+  row += list_end;// + std::string(",");
```
Comment (jkottiku) [inline src/rvsliblogger.cpp:366]: Remove commented code.

**Review by jkottiku — APPROVED** [review]

**Comment by jkottiku** [comment]
1. Please confirm if the new json files parse properly use online json parser.
2. Json file key-value overall refactoring needs to planned.

File: src/rvsliblogger.cpp (hunk 1 above)
Comment (johnvuongAMD) [reply src/rvsliblogger.cpp:355]: When will the fix be implemented? I just checked rvsliblogger.cpp and I can’t find the changes.

**Comment by manoj-freyr** [comment]
> 1. Please confirm if the new json files parse properly use online json parser.
> 2. Json file key-value overall refactoring needs to planned.

I used jq tool, with default filter to parse logs. Parsing was fine. Online tool sometimes removes last comma and parses it correctly even when jq raises error.

**Pull request merged**